.travis.yml

```diff
@@ -2,32 +2,41 @@ sudo: false
 
 language: php
 
-php:
-  - 7.1
-  - 7.2
-
 cache:
   directories:
-    - $HOME/.composer/cache/files
+    - $HOME/.composer/cache
 
 env:
   global:
-    - SYMFONY_DEPRECATIONS_HELPER=weak
-    - SYMFONY_VERSION=3.4.*
+    - SYMFONY_DEPRECATIONS_HELPER="weak_vendors"
 
 matrix:
-  include:
-    - env: SYMFONY_VERSION=4.0.*
   fast_finish: true
-
+  include:
+    - php: 7.1
+      env: SYMFONY_VERSION=4.0.*
+    - php: 7.1
+      env: SYMFONY_VERSION=3.4.*
```

Comment:
i don't think we need to build with symfony 3.4 on more than 1 version of php. 7.2 is enough.